streams/src/test/java/org/apache/kafka/streams/tools/StreamsResetterTest.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.kafka.streams.tools;
 
-import kafka.tools.StreamsResetter;
+import org.apache.kafka.tools.StreamsResetter;
```

Comment:
I think this test belongs to the tools module, where the class under test lives.

Reply:
Yeah even I had thought so. But the original layout was also similar (i.e StreamsResetter was under tools and the test lived here) so I didn't change it. @cadonna , WDYT?

Reply:
I would also move it to the tools module. 
@mjsax Was there a specific reason not to have the test in the same module as the `StreamsResetter`?

Reply:
Got it @cadonna . Would wait for the response. Even I think it can move but let's see what guozhang and Mathias have to say.

Reply:
I would not wait for the answer. We can do that in a follow-up PR. Could you open a ticket for the move?

Reply:
Thanks @cadonna , here's the follow up ticket: https://issues.apache.org/jira/browse/KAFKA-14851